README.md

```diff
@@ -76,18 +75,18 @@ The app exposes these endpoints:
 
 An app that uses a Tanzu GemFire service instance may be
 located in one of three locations,
-as specified in [The App's Location](https://docs.pivotal.io/p-cloud-cache/2-2/architecture.html#AppLocation).
+as specified in [The App's Location](https://techdocs.broadcom.com/us/en/vmware-tanzu/data-solutions/tanzu-gemfire-on-cloud-foundry/2-1/gf-cf/content-architecture.html#AppLocation).
 This app demonstrates all three possibilities of app location
 using Spring profiles.
 
 In addition, the app may be run locally,
-without having a Geode or Tanzu GemFire service instance.
+without having a Tanzu GemFire service instance.
 All four running environments are described.
 
 ## Run the App in a Local Environment
```

Comment:
I think we need to review this section. 
1. EnableClusterAware no longer creates a local cache, so users will need to spin up a GemFire cluster locally.
2. The link here is incorrect. It should link to the new documentation, not Spring Boot Data Geode.